websocket/backend/Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'rack-cors'
```

Comment:
What is this for?

Reply:
`rack-cors `provides support for Cross-Origin Resource Sharing (CORS) for Rack compatible web applications. So we can make requests from one website to another (frontend to backend with different URLs).

Reply:
oh yes. I forgot you have to add in separately. ok